### ISSUE TITLE
WPP not found for version 2.3000.1011681023

### ISSUE THREAD

**Issue by abhyudayasrinet** [wppconnect-team/wa-js#1699]
## Description

I've injected the wpconnect.js file successfully but the WPP variable doesn't load on the window.
Works fine on older whatsapp versions like 2.2408.1

## Environment

- **WhatsApp version:** 2.3000.1011681023
- **WA-JS version(s):** 2.28.1
- **Browser:** Chrome 122.0.6261.69
- **OS:** Windows + Mac

## Your Code

```
// content file
var s = document.createElement('script');
s.src = chrome.runtime.getURL('scripts/wa-inject.js');
s.onload = function () {
  console.log("[content.js] wa-inject.js injected");
  this.remove();
};
(document.head || document.documentElement).appendChild(s);

// inject.js
function waitForWPP() {
    if (typeof window.WPP !== "undefined") {
        //variable exists, do what you want
        console.log("[inject.js] WPP exists");
        // do something
    }
    else {
        console.log("[inject.js] WPP not found");
    }
}

```

**Comment by marcelo386** [comment]
![IMG-20240304-WA0021](https://github.com/wppconnect-team/wa-js/assets/69150213/8c48426f-c9f8-4c1d-9f38-842e53680b2a)
Bug getMessage

**Comment by marcelo386** [comment]
![IMG-20240304-WA0026](https://github.com/wppconnect-team/wa-js/assets/69150213/fd57a812-f47e-436f-81c6-9be94158795f)
![IMG-20240304-WA0025](https://github.com/wppconnect-team/wa-js/assets/69150213/de48f863-a44d-4410-9233-f30f7c171a7d)
Bugs New Version WA-JS

**Comment by abhyudayasrinet** [comment]
tried building the current `whatsapp_2.3000.x` branch and running on it.

Got this error:
![image](https://github.com/wppconnect-team/wa-js/assets/5520691/5871f621-e251-4a0c-98e8-ad1dfde15b63)

**Comment by above7sky** [comment]
> ## Description
> I've injected the wpconnect.js file successfully but the WPP variable doesn't load on the window. Works fine on older whatsapp versions like 2.2408.1
> 
> ## Environment
> * **WhatsApp version:** 2.3000.1011681023
> * **WA-JS version(s):** 2.28.1
> * **Browser:** Chrome 122.0.6261.69
> * **OS:** Windows + Mac
> 
> ## Your Code
> ```
> // content file
> var s = document.createElement('script');
> s.src = chrome.runtime.getURL('scripts/wa-inject.js');
> s.onload = function () {
>   console.log("[content.js] wa-inject.js injected");
>   this.remove();
> };
> (document.head || document.documentElement).appendChild(s);
> 
> // inject.js
> function waitForWPP() {
>     if (typeof window.WPP !== "undefined") {
>         //variable exists, do what you want
>         console.log("[inject.js] WPP exists");
>         // do something
>     }
>     else {
>         console.log("[inject.js] WPP not found");
>     }
> }
> ```

How did you manage to install version 3.0 of WhatsApp for iPhone?

**Comment by above7sky** [comment]
Today I managed to get to this error.

Using WhatsApp Web, I went to settings and then help and activated the Beta version and when updating the screen, the inject error occurred.

I tried other scripts that use inject the same problem occurred.

The active beta version was the same as mentioned above 2.3xxx.

Problems in sight :(
![error](https://github.com/wppconnect-team/wa-js/assets/8179748/3b02474b-b2e1-41a1-87ef-f369845ecc49)

The attached image shows that we have a new structure.
